charts/evm-rollup/values.yaml

```diff
@@ -92,6 +100,10 @@ genesis:
       value:
         balance: "0"
         code: "0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe03601600081602082378035828234f58015156039578182fd5b8082525050506014600cf3"
+    - address: "0x0000000000000000000000000000000000000089"  ## oracle contract
```

Comment:
Generally speaking should probably add on the dev values instead of on the core chart as we do with other contract values

Reply:
This appears to already exist in the dev values.  I'll just delete from here.

Reply:
Done in [60a150b](https://github.com/astriaorg/astria/pull/2085/commits/60a150b91b5d990cf9d0e9a6b85b8a788c40cbac).